README.md

```diff
@@ -4,40 +4,40 @@
 
 ## Installing
 
-* Make sure you're running at least Node 16 - you may want to avail yourself of a tool like [nvm](https://nvm.sh)
-* Run `yarn` to install dependencies
-* Run `yarn docker:start` to start docker
-* Run `yarn build` to build things locally
-* Run `yarn db:client:generate` to generate the Prisma client
-* Run `yarn db:migrate` to create a migration
-* (Optional) Run `yarn db:seed:testing` to load some sample data into the database
+- Make sure you're running at least Node 20 - you may want to avail yourself of a tool like [nvm](https://nvm.sh)
```

Comment:
VSCode's auto-formatter wanted to do this, I guess. In Markdown, `*` and `-` _are_ equivalent, so 🤷